### PR TITLE
make customParams (for mapfish print WMS url setup) parametrizeable

### PR DESCRIPTION
**Pull request by jwkaltz**
make customParams (for mapfish print WMS url setup) parametrizeable, instead of hard-coded as they are now.

### CODE REVIEW

**Comment by ralphschulerSZ** [comment]
Hi Wolfgang, I tested your code and it works fine here.

File: pyramid_oereb/contrib/print_proxy/mapfish_print.py [hunk 1]
```diff
@@ -131,6 +131,29 @@ def __call__(self, value, system):
...
+        :rtype: list
+        """
+        result = {}
+        wms_url_params = Config.get('print', {}).get('wms_url_params', False)
+        if wms_url_params:
+            log.debug("get_wms_url_params() read configuration {}".format(wms_url_params))
+            if isinstance(wms_url_params, dict):
+                result = wms_url_params
+            else:
+                log.warning("get_wms_url_params() ignoring unaccepted configuration value {}"
```
Comment (pfirpfel) [inline pyramid_oereb/contrib/print_proxy/mapfish_print.py]: Shoudln't it still return the default values in this case?

Comment (jwkaltz) [reply pyramid_oereb/contrib/print_proxy/mapfish_print.py]: IMO in that case it is better to return nothing, this way it is more likely that the user (the bad configurator) will notice something is wrong.

Comment (pfirpfel) [reply pyramid_oereb/contrib/print_proxy/mapfish_print.py]: Ok for me

**Pull request merged**